.eslintrc.js

```diff
@@ -4,6 +4,10 @@ module.exports = {
     {
       env: { mocha: true },
       files: ['test/**/*.js'],
+      parserOptions: { sourceType: 'module' },
+      rules: {
+        'n/no-extraneous-import': 'off',
```

Comment:
I assume because we use a "fake" `package.json` file to force ES modules for the `test` folder. Can probably be deleted once we use `"type": "module"` in the root `package.json` file.